uobtheatre/bookings/test/test_mutations.py

```diff
@@ -21,6 +22,19 @@
 from uobtheatre.venues.test.factories import SeatFactory, SeatGroupFactory
 
 
+def add_ticket_to_booking(booking):
```

Comment:
Can we move this into bookings/test/factories

Reply:
Maybe with some (optional?) arguments?

Reply:
if we need to extend later with arguments we can, I think for now this should be enough, its mainly just to make it a valid booking